src/utils/common-utils.ts

```diff
@@ -222,3 +87,10 @@ export const isFCPSupported = () => {
 export const isLongTaskSupported = () => {
     return PerformanceObserver.supportedEntryTypes.includes('longtask');
 };
+
+export const isPutRumEventsCall = (url: string, endpointHost: string) => {
+    const pathRegex =
+        /.*\/application\/[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}\/events/;
```

Comment:
question: This looks like it accounts for proxy endpoints, correct? Do we have unit tests for both proxy and non-proxy enpoints?

Reply:
Good catch, this is old logic but I will deep dive on this topic.

Reply:
I think the preexisting logic for isPutRumEvents() is wrong. For example, a valid url is `https://dataplane.rum.us-west-2.amazonaws.com/appmonitors/265d8f93-d021-44e1-b0c5-41fcc660842f`, which would fail the validation. Path should be `appmonitors` instead of `application`. And there is no `events` path. This deserves its own PR.